spinegeneric/config/specs.json

```
@@ -0,0 +1,444 @@
+                    "FlipAngle":15
+                }
+            }
+        },
+        "GE":{
+            "DISCOVERY_MR750":{
+                "T1w":{
+                    "InversionTime": 1,
+                    "FlipAngle":9
+                },
+                "T2w":{
+                    "RepetitionTime":2.5,
+                    "EchoTime":0.120
+                },
+                "T2star":{
+                    "RepetitionTime":0.6,
+                    "FlipAngle":30
+                },
+                "MTon_MTS":{
+                    "RepetitionTime":0.035,
+                    "EchoTime":0.004,
+                    "FlipAngle":9
+                },
+                "MToff_MTS":{
+                    "RepetitionTime":0.035,
+                    "EchoTime":0.004,
+                    "FlipAngle":9
+                },
+                "T1w_MTS":{
+                    "RepetitionTime":0.015,
+                    "EchoTime":0.004,
+                    "FlipAngle":15
+                }
+            },
+            "DISCOVERY_MR750w":{
+                "T1w":{
+                    "InversionTime": 1,
+                    "FlipAngle":9
+                },
+                "T2w":{
+                    "RepetitionTime":2.5,
+                    "EchoTime":0.120
+                },
+                "T2star":{
+                    "RepetitionTime":0.6,
+                    "FlipAngle":30
+                },
+                "MTon_MTS":{
+                    "RepetitionTime":0.035,
+                    "EchoTime":0.004,
+                    "FlipAngle":9
+                },
+                "MToff_MTS":{
+                    "RepetitionTime":0.035,
+                    "EchoTime":0.004,
+                    "FlipAngle":9
+                },
+                "T1w_MTS":{
+                    "RepetitionTime":0.015,
+                    "EchoTime":0.004,
+                    "FlipAngle":15
+                }
+            },
+            "Signa_HDxt":{
+                "T1w":{
+                    "InversionTime": 1,
+                    "FlipAngle":9
+                },
+                "T2w":{
+                    "RepetitionTime":2.5,
+                    "EchoTime":0.120
+                },
+                "T2star":{
+                    "RepetitionTime":0.6,
+                    "FlipAngle":30
+                },
+                "MTon_MTS":{
+                    "RepetitionTime":0.035,
+                    "EchoTime":0.004,
+                    "FlipAngle":9
+                },
+                "MToff_MTS":{
+                    "RepetitionTime":0.035,
+                    "EchoTime":0.004,
+                    "FlipAngle":9
+                },
+                "T1w_MTS":{
+                    "RepetitionTime":0.015,
+                    "EchoTime":0.004,
+                    "FlipAngle":15
+                }
+            },
+            "SIGNA_PET_MR":{
+                "T1w":{
+                    "InversionTime": 1,
```

to keep the convention
```suggestion
                    "InversionTime":1,
```

---

Fixed in https://github.com/spine-generic/spine-generic/pull/198/commits/a0cc957369e53c27c21155e612e84d51e8d571e2